app/Http/Middleware/Authenticate.php

```diff
@@ -15,7 +15,7 @@ class Authenticate extends Middleware
     protected function redirectTo($request)
     {
         if (! $request->expectsJson()) {
-            return route('login');
```

Comment:
route names zijn lowercase

Reply:
Dit heb ik niet geprogrammeerd, geen idee waarom dit in deze pull request besproken wordt.